Task_TheFlexGame/index.css

```diff
@@ -1 +1,73 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Responsive Layout</title>
+  <style>
+    body {
+      margin: 0;
+      font-family: Arial, sans-serif;
+      display: flex;
+    }
+
+    .menu {

```

Comment:
The width is given as a percentage. It is recommended to use a fixed width value for better responsiveness.